src/vivarium_gates_nutrition_optimization/constants/paths.py

```diff
@@ -5,7 +5,7 @@
 
 BASE_DIR = Path(vivarium_gates_nutrition_optimization.__file__).resolve().parent
 
-ARTIFACT_ROOT = Path(f"/share/costeffectiveness/artifacts/{metadata.PROJECT_NAME}/")
+ARTIFACT_ROOT = Path(f"/team/simulation_science/pub/models/vivarium_gates_nutrition_optimization/artifacts/{metadata.PROJECT_NAME}/")
```

Comment:
Fine to keep, but the path should be f"/mnt/team/simulation_science/pub/models/{metadata.PROJECT_NAME}/artifacts/".